### ISSUE TITLE
Occasional mysterious ReadHistory failure

### ISSUE THREAD

**Issue by Frenzie** [koreader/koreader#3097]
* KOReader version: I've seen it a couple of times, most recently 7376c57
* Device: Travis

#### Issue
```
Failure → spec/front/unit/readhistory_spec.lua @ 226

ReadHistory module should be able to remove three items

spec/front/unit/readhistory_spec.lua:40: Expected objects to be the same.

Passed in:

(string) 'a'

Expected:

(string) 'd'
```

Restarting the build generally works.

#### Steps to reproduce
Unknown.

**Comment by poire-z** [comment]
OK, there are a few more tests (#2985, #3380) that could benefit from a usleep(1000000) between the addItem().

**Comment by Frenzie** [comment]
Oh, now I understand your earlier remark. I'm not entirely sure if the tests are the right place to "fix" this. Pro Starcraft gamers perform 10 actions per second.

**Comment by poire-z** [comment]
Must have been solved by #6991.
